### PR TITLE
Moto gp

### PR DESCRIPTION
**Pull request by JohnteFresh**
Add MotoGP main page
Add about page
Create Drivers section in JavaScript

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+  </main>
+  <footer class="hidden">
+    <img class="footer-logo" src="icons/MotoImg/Fédération_Internationale_de_Motocyclisme_logo.svg.png" alt="" class="footer-desktop-img">
+    <div>
+      <p>@2023 Dorna Sports SL <br> All rights reserved <br> All trademarks are the property of their respective owners.</p>
+      <p class="privacy-policy hidden">For details about how we use your information, please see our <a href="#">Privacy Policy</a></p>
+      <p class="cookies-terms hidden"><a href="#">Terms & Conditions</a> | <a href="#">Cookies</a></p>
+      </div>
+  </footer>
+<script src="./JS/index.js"></script>
```
Comment (ShadyShawkat) [inline about.html]: - Please make sure to stick as much as possible, which includes:
   - [x] Adjusting the images section on the top of the images below to be on one row in the desktop view to match the given template.
   - [x] Removing the "Partners" section in the "About" page to reflect the given design.
   - [x] Changing the background of the footer to match the original design.

| Your design | Original design |
| --- | --- |
| image | image |

File: index.html [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+    <!-- <section id="moto-home">
+      <div class="img-area"></div>
+      <div class="intro">
+        <h3><img src="icons/MotoImg/Flag_of_the_United_Kingdom_(1-2).svg.png" alt=""></h3>
+        <h1>MotoGP Rider Fan Parade</h1>
+        <h2>
+          #BritishGP
+          <span>LIVE:Don't miss any of the Sunday's action at the British GP!</span>
+        </h2>
+        <button>WATCH NOW</button>
```
Comment (ShadyShawkat) [inline index.html]: - [x] Kindly remove unwanted commented code to make your project professional.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+      <ul class="desktop-nav">
+          <!-- <li><a href="index.html" class="page-links">Home</a></li> -->
+          <li><a href="about.html" class="page-links">About</a></li>
+          <li><a href="#drivers" class="page-links">Our Riders</a></li>
+          <li><a href="#race-days" class="page-links">Races</a></li>
+          <li><a href="#https://www.formula1.com/en/latest.html" class="page-links">News</a></li>
+          <li><a href="https://www.formula1.com/en/latest/tags.race-calendar.2Sa4DEdBSUMcOqa8aYceEe.html" class="page-links campaign" id="campaign">F1 Campaign</a></li>
+      </ul>
+    </nav>
+  </header>
```
Comment (ShadyShawkat) [inline index.html]: - [x] Please adjust the spacing between the logo of the navbar (on the left) and the navbar menu elements (on the right).
- [x] Kindly add some padding to the sides of the navbar, as currently, the logo and navbar elements are too close to the edge of the screen.

| Your design | Original design |
| --- | --- |
| image | image |

File: about.html [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+        <div class="vid-2">
+          <a href="#">MotoGP Grand Prix<h3>2022</h3></a>
+        </div>
+        <div class="vid-3">
+          <a href="#">MotoGP Grand Prix<h3>2021</h3></a>
+        </div>
+      </div>
+    </section>
+
+    <section id="sponsors" class="hidden">
```
Comment (ShadyShawkat) [inline about.html]: - [x] Kindly add some spacing to the sides of the following sections, as currently, they are too close to the edge of the screen.

| Your design | Original design |
| --- | --- |
| image | image |

File: index.html [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+    <section id="moto-home">
+      <q class="subtitle">Welcome to the new MotoGP beta website!</q>
+  
+      <h1 class="cocogoose">Monster Energy British grand Prix</h1>
+      <h2 class="year">2023</h2>
+      <p class="annotation">BritishGP <span>TALKING POINTS</span> MotoGP News ...</p>
+      <q class="news">I've always said Aleix has the biggest balls in the paddock</q>
+      <p class="date">2023.08.06(SUN)</p>
+      <p class="address">@MotoGP, British Grand Prix</p>
+    </section>
```
Comment (ShadyShawkat) [inline index.html]: - [x] Kindly add some spacing around the following section, to match the given design.
- [x] Please also increase the font size of the title to match the given design.

| Your design | Original design |
| --- | --- |
| image | image |

File: index.html [hunk 6]
```diff
@@ -0,0 +1,150 @@
...
+      </article>
+      <a href="#" class="see-races">SEE ALL THE RACES</a>
+      <a href="#" class="tickets">BUY TICKETS NOW!</a>
+    </section>
+  <!-- featured speakers -->
+
+    <section id="drivers">
+      <h2>Our Racers</h2>
+      <hr>
+    </section>
```
Comment (ShadyShawkat) [inline index.html:107]: - [x] Kindly add some spacing around the following section, to match the given design.

| Your design | Original design |
| --- | --- |
| image | image |

File: index.html [hunk 7]
```diff
@@ -0,0 +1,150 @@
...
+      <article id="technical">
+        <img src="icons/MotoImg/customer-service.png" alt="support icon">
+          <h3>Technical</h3>
+          <p>A sneak peak into the teams' operations and Meet the team prinicpals of the top teams ...</p>
+      </article>
+
+      <article id="champion">
+        <img src="icons/MotoImg/trophy.png" alt="trophy-icon">
+            <h3>Champions</h3>
+            <p>The F1 trophy will be on display along with some legends and get your own special replica ...</span></p>
```
Comment (ShadyShawkat) [inline index.html:97]: - [x] Kindly add some spacing around the following section, to match the given design.

| Your design | Original design |
| --- | --- |
| image | image |

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @JohnteFresh, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- All linters are passing ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: about.html (hunk 1 above)
Comment (JohnteFresh) [reply about.html]: Resolved according to the review.

File: index.html (hunk 3 above)
Comment (JohnteFresh) [reply index.html]: Solved

File: about.html [hunk 8]
```diff
@@ -0,0 +1,113 @@
...
+    <img class="footer-logo" src="icons/MotoImg/Fédération_Internationale_de_Motocyclisme_logo.svg.png" alt="" class="footer-desktop-img">
+    <div>
+      <p>@2023 Dorna Sports SL. All rights reserved <br> All trademarks are the property of their respective owners.</p>
+      <p class="privacy-policy hidden">For details about how we use your information, please see our <a href="#">Privacy Policy</a></p>
+      <p class="cookies-terms hidden"><a href="#">Terms & Conditions</a> | <a href="#">Cookies</a></p>
+      </div>
+  </footer>
+<script src="./JS/index.js"></script>
+</body>
+</html>
```
Comment (usorfaitheloho) [inline about.html:113]: As suggested by the previous reviewer, please ensure to stick to the design as closely as possible .  kindly adjust the images section on the top of the images below to be on one row in the desktop view to match the given template. You currently have three images, meanwhile the images are supposed to be just two 👍 

Your design :
image

Design given :

File: index.html [hunk 9]
```diff
@@ -0,0 +1,133 @@
...
+      </div>
+
+      <ul class="desktop-nav">
+          <li><a href="about.html" class="page-links-1">About</a></li>
+          <li><a href="#drivers" class="page-links">Our Riders</a></li>
+          <li><a href="#race-days" class="page-links">Races</a></li>
+          <li><a href="#https://www.formula1.com/en/latest.html" class="page-links">News</a></li>
+          <li><a href="https://www.formula1.com/en/latest/tags.race-calendar.2Sa4DEdBSUMcOqa8aYceEe.html" class="page-links campaign" id="campaign">F1 Campaign</a></li>
+      </ul>
+    </nav>
```
Comment (usorfaitheloho) [inline index.html]: [Optional] It'd be nice to adjust the spacing between the nav items as your design is quite different from the design given in that aspect. The spacing between your nav items is alot and not so nice aesthetically 😄 . Also your nav color for some of the items is blue,which is different from the color in the design

Your design :

image

Figma design :
image

File: index.html [hunk 10]
```diff
@@ -0,0 +1,133 @@
...
+    <section id="moto-home">
+      <q class="subtitle">Welcome to the new MotoGP beta website!</q>
+  
+      <h1>Monster Energy British grand Prix</h1>
+      <h2 class="year">2023</h2>
+      <p class="annotation">BritishGP <span>TALKING POINTS</span> MotoGP News ...</p>
+      <q class="news">I've always said Aleix has the biggest balls in the paddock</q>
+      <p class="date">2023.08.06(SUN)</p>
+      <p class="address">@MotoGP, British Grand Prix</p>
+    </section>
```
Comment (usorfaitheloho) [inline index.html:65]: I'd suggest you reduce the width of the image here as it looks too close to the margin and it's different from our deisgn given 😄 

Figma design :
image

Your design :
image

File: index.html (hunk 6 above)
Comment (usorfaitheloho) [reply index.html:107]: As suggested by the previous reviewer, kindly add some spacing between the speakers as they are too close to each other and not too similar to the deign given 😄 

Your design :
image

Design given :
image

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
Hi @JohnteFresh  :thumbsup:  ,

This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

There are still some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Good use of JavaScript, HTML&CSS best practices ✔️ 
-  Well-structured files ✔️
<br/>
<br/>
<br/>

 ## Required Changes  :recycle: 

_Check the comments under the review._ 

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 10 above)
Comment (JohnteFresh) [reply index.html:65]: Done

**Review by Gambit142 — APPROVED** [review]
Hi @JohnteFresh 👋,

## Status: Approved 🟢   

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

### Highlights
- Setup linters correctly ✔️ 
- Application has two pages ✔️ 
- Created a mobile menu ✔️ 
- Rider's section is rendered dynamically ✔️ 

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**